### ISSUE TITLE
Issue with `to_js_value` & `from_js_value` - building for ia32 on x64 windows

### ISSUE THREAD

**Issue by kyleshay** [napi-rs/napi-rs#896]
Hello! I'm trying to use the `to_js_value` & `from_js_value` to parse an object input to a rust struct. It works fine on 64bit but when I build it for 32 and run the program by calling either of the following from node — the program just shuts down.
 `require('./index.js').parse_obj({id: 1})`
 `require('./index.js').to_obj()`

The node file is created on a windows machine with: `napi build --platform --target i686-pc-windows-msvc`

lib.rs
```rust
#![deny(clippy::all)]

#[macro_use]
extern crate serde_derive;

#[macro_use]
extern crate napi_derive;

use napi::{CallContext, JsNumber, JsObject, JsUnknown, Result};

#[module_exports]
fn init(mut exports: JsObject) -> Result<()> {
    exports.create_named_method("parse_obj", parse_obj)?;
    exports.create_named_method("to_obj", to_obj)?;
    Ok(())
}

#[derive(Deserialize)]
pub struct Props {
    pub id: u32,
}

#[js_function(1)]
pub fn parse_obj(ctx: CallContext) -> Result<JsNumber> {
    let props = ctx.get::<JsObject>(0)?;
    let props: Props = ctx.env.from_js_value(props)?;
    ctx.env.create_uint32(props.id)
}

#[js_function(0)]
pub fn to_obj(ctx: CallContext) -> Result<JsUnknown> {
    ctx.env.to_js_value(&Props { id: 1 })
}
```
cargo.toml
```toml
[dependencies]
napi = { version = "1", features = ["serde-json"] }
napi-derive = "1"
serde = "1"
serde_derive = "1"
```

**Comment by Brooooooklyn** [comment]
This issue was fixed in `napi@2.0.0-alpha`, could you have a try with the 2.0?

**Comment by kyleshay** [comment]
Nice, confirmed it's working in alpha! I'm now seeing an issue with building classes though when building for ia32.

I was not able to get this example to run with ia32 (worked fine with x64 and I could build it on my mac as well.)
https://github.com/napi-rs/napi-rs/blob/main/examples/napi/src/class.rs

Happy to create a separate issue for that one though.
